src/components/Collection/ListViewItem.tsx

```diff
@@ -29,6 +30,8 @@ import type { CollectionGroupType } from '@/core/types/api/collection';
 import type { SeriesSizesFileSourcesType, SeriesType } from '@/core/types/api/series';
 import type { WebuiGroupExtra } from '@/core/types/api/webui';
 
+dayjs.extend(advancedFormat);
```

Comment:
probably makes sense to put it in util.ts and export the extended dayjs

Reply:
Actually... it doesn't need to be exported. It does some magic.
I put it in `util.ts` and it just started working everywhere